lib/activerecord/cte/core_ext.rb

```diff
@@ -6,10 +6,6 @@ module Querying
   end
 
   module WithMerger
-    def normal_values
```

Comment:
@dark-panda when I looked at the 6.0.x internals, this made it so Rails automatically did some things. What steps that Rails was doing are not not being performed as a result of not including `:with` here? Do we need to manually perform those steps?

I believe `normal_values` was removed in 6.1.x, so we need to consider how this affects both versions.

Reply:
`#normal_values` is still in as of the current 6-1-stable:

https://github.com/rails/rails/blob/6-1-stable/activerecord/lib/active_record/relation/merger.rb#L58